unlock-app/src/components/interface/locks/Settings/forms/LockManagerForm.tsx

```diff
@@ -304,10 +318,14 @@ export const LockManagerForm = ({
           onSubmit={handleSubmit(onAddLockManager)}
         >
           <div className="flex flex-col gap-2">
-            <span className="text-base text-brand-dark">
-              Add manager, please enter the wallet address of theirs.
-            </span>
-            <Input disabled={disableInput} {...register('manager')} />
+            <AddressInput
+              withIcon
+              name="manager"
+              label="Add manager, please enter the wallet address of theirs."
+              description="Enter a wallet address or an ens name"
+              web3Service={web3Service}
+              localForm={localForm}
```

Comment:
- We can manage `web3Service ` inside the component to make sure we use the correct web3service and is specific  `await web3Service.resolveName(address)`, because now the type is any, potentially fail because is not a required prop.
- same for `localForm` maybe
@searchableguy @julien51  wdyt?

Reply:
Oh nice catch, I can make the prop required. I initially thought of defining `web3Service` inside the UI package but I don't think it's that necessary. Cause other than the `AddressInput` component, no other component will be using it. So I think it will be much easier if we pass it down as a prop. @kalidiagne

Reply:
Regarding the `localForm`, @searchableguy did suggest using `useFormContext` but for some reason, I couldn’t get it to work. Maybe because our UI package lives in a separate repo, that's why react-hook-form is not aware of the `context`, not sure... 

But as I said earlier, I have seen people follow the same approach, passing down the form hook as a prop. For example, you can check [this](https://github.com/raid-guild/design-system/blob/main/src/components/forms/Input/Input.tsx#L42) out. @kalidiagne

Reply:
Hey, @kalidiagne @searchableguy @julien51 Are we good to move forward with this?